### PR TITLE
More javadocs

### PR DESCRIPTION
**Pull request by g4s8**
#1126 - added javadocs for some classes

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1128](https://github.com/zerocracy/farm/issues/1128) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1128](https://github.com/zerocracy/farm/issues/1128) is assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1611606822); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/zerocracy/entry/Pings.java [hunk 1]
```diff
@@ -36,6 +36,10 @@
 
 /**
  * Pings.
+ * <p>
+ * This class sends broadcasts (pings) in the system,
+ * it submit new claim with type 'ping'
+ * every minute, hour and day.
```
Comment (krzyk) [inline src/main/java/com/zerocracy/entry/Pings.java]: @g4s8 there should be a single empty comment line between the description and tags (this is the convention we have in the projects and it is also existins in other open source projects including OpenJDK)

File: src/main/java/com/zerocracy/pmo/Blanks.java [hunk 2]
```diff
@@ -25,6 +25,9 @@
 
 /**
  * Blanks.
+ * <p>
+ * User metric which represents the amount of bugs user reported,
+ * which were not counted as bugs by project architects.
```
Comment (krzyk) [inline src/main/java/com/zerocracy/pmo/Blanks.java]: @g4s8 there should be a single empty comment line between the description and tags

File: src/main/java/com/zerocracy/pmo/Speed.java [hunk 3]
```diff
@@ -27,6 +27,8 @@
 
 /**
  * Speed of delivery.
+ * <p>
+ * User metric which represents how much time user spend to finish a job.
```
Comment (krzyk) [inline src/main/java/com/zerocracy/pmo/Speed.java]: @g4s8 there should be a single empty comment line between the description and tags

File: src/main/java/com/zerocracy/entry/Pings.java (hunk 1 above)
Comment (krzyk) [inline src/main/java/com/zerocracy/entry/Pings.java]: @g4s8 doesn't `every minute` imply also `every hour` and `every day`? Or it is an option in this class than one can choose? In either case I think javadoc should be `every minute.` or `every minute or every hour or every day`.

File: src/main/java/com/zerocracy/pmo/Pmo.java [hunk 4]
```diff
@@ -25,7 +25,13 @@
 import org.cactoos.scalar.SolidScalar;
 
 /**
- * PMO.
+ * PMO (project management office).
+ * <p>
+ * PMO is a project with a special status.
+ * It has its own set of items, own XSD schemas, everything on its own.
+ * We keep system information there, like list of all users (people.xml),
+ * list of all projects (catalog.xml),
```
Comment (krzyk) [inline src/main/java/com/zerocracy/pmo/Pmo.java]: @g4s8 the next line can be joined with current one

**Comment by g4s8** [comment]
@krzyk fixed, please look again

**Comment by krzyk** [comment]
@g4s8 thanks

**Comment by krzyk** [comment]
@rultor merge

**Comment by krzyk** [comment]
@g4s8 I think you need to confirm (until https://github.com/zerocracy/farm/issues/1093 is done)

**Comment by g4s8** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by krzyk** [comment]
@g4s8 maybe it would be good to fix it - https://github.com/zerocracy/farm/issues/1032

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1128](https://github.com/zerocracy/farm/issues/1128) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +20 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**